_layouts/page_footer.html

```diff
@@ -1,7 +1,77 @@
+  </symbol>
+</svg>
 <footer class="page-footer container">
+  <section class="synopsis grid-container">
+    <div class="g-row">
+      <div class="g-col-3">
+        <h3>Connect</h3> <!-- (social media) -->
+        <nav aria-label="Connect">
+          <ul class="connect" aria-label="Connect">
+            <li><a href="https://es.discourse.group/">
+                <span class="icon">
+                  {% octicon comment-discussion height:24 aria-label:Discourse %}
+                </span>
+                <span>
+                  Discourse
+                </span>
+              </a></li>
+            <li><a href="https://github.com/tc39/">
+                <span class="icon">
+                  {% octicon mark-github height:24 aria-label:GitHub %}
+                </span>
+                <span>
+                  GitHub
+                </span>
+              </a></li>
+            <li><a href="http://twitter.com/tc39" target="_blank">
+                <span class="icon">
+                  <svg width="21" height="18" class="twitter-logo" aria-label="Twitter">
+                    <use xlink:href="#twitterIcon"></use>
+                  </svg>
+                </span>
+                <span>
+                  Twitter
+                </span>
+              </a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="g-col-3">
+        <h3>Specs</h3> <!-- (standards, how to spec.) -->
+        <ul>
+          <li><a href="https://tc39.es/ecma262/">ECMA-262, ECMAScript</a></li>
+          <li><a href="https://tc39.es/ecma402/">ECMA-402, Intl API</a></li>
+          <li><a href="https://timothygu.me/es-howto/" target="_blank">Reading the Spec</a></li>
+        </ul>
+      </div>
+      <div class="g-col-3">
+        <h3>Proposals</h3> <!-- (list of proposals and their state) -->
+        <ul>
+          <li><a href="https://github.com/tc39/proposals/blob/master/stage-0-proposals.md">Stage 0</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/stage-1-proposals.md">Stage 1</a></li>
+          <li><a href="https://github.com/tc39/proposals#stage-2">Stage 2</a></li>
+          <li><a href="https://github.com/tc39/proposals#stage-3">Stage 3</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/finished-proposals.md">Stage 4 (Finished)</a></li>
+          <li><a href="https://github.com/tc39/proposals/blob/master/inactive-proposals.md">Inactive</a></li>
```

Comment:
I would like to put the contents of these Markdown files into data structures.

Reply:
what do you mean here? do you mean a rendered view on the website?